src/ckb/channel.rs

```diff
@@ -125,37 +129,45 @@ pub struct ChannelCommandWithId {
     pub command: ChannelCommand,
 }
 
-pub const DEFAULT_FEE_RATE: u64 = 0;
-pub const DEFAULT_COMMITMENT_FEE_RATE: u64 = 0;
+pub const DEFAULT_FEE_RATE: u64 = 1_000;
+pub const DEFAULT_COMMITMENT_FEE_RATE: u64 = 1_000;
 pub const DEFAULT_MAX_TLC_VALUE_IN_FLIGHT: u128 = u128::MAX;
 pub const DEFAULT_MAX_ACCEPT_TLCS: u64 = u64::MAX;
 pub const DEFAULT_MIN_TLC_VALUE: u128 = 0;
 pub const DEFAULT_TO_LOCAL_DELAY_BLOCKS: u64 = 10;
-pub const DEFAULT_UDT_MINIMAL_CKB_AMOUNT: u64 = 200 * 100_000_000; // 200 CKB
 
 #[derive(Debug)]
 pub struct TxUpdateCommand {
     pub transaction: Transaction,
 }
 
-#[derive(Debug)]
-pub struct TxCompleteCommand {}
+pub struct OpenChannelParameter {
```

Comment:
Why there's no `reserve_ckb_amount` in `OpenChannelParameter`?

Reply:
`reserve_ckb_amount` currently is not specified by rpc, it's determined by hardcoded parameter:

https://github.com/nervosnetwork/cfn-node/pull/69/files#diff-14d3f58d612f44a1c17f7411a4ee8d3fa521613115bd6b221e29ceb730c98a0cR992-R996

Maybe we can add option so that users can specify them, but check these parameters are larger than the Default minimal ones?